exo-create/main_test.go

```
@@ -0,0 +1,162 @@
+			return err
+		}
+		return run(cwd, command)
+	})
+
+	s.Step(`^entering into the wizard:$`, func(table *gherkin.DataTable) error {
+		for _, row := range table.Rows[1:] {
+			if err := enterInput(row); err != nil {
+				return err
+			}
+		}
+		defer in.Close()
+		return nil
+	})
+
+	s.Step(`^running "([^"]*)" in the terminal$`, func(command string) error {
+		appDir = path.Join(cwd, "tmp")
+		if err := testHelpers.EmptyDir(appDir); err != nil {
+			return err
+		}
+		return run(cwd, command)
+	})
+
+	s.Step(`^waiting until I see "([^"]*)" in the terminal$`, func(expectedText string) error {
+		if err := cmd.Wait(); err != nil {
+			return err
+		}
+		childOutput = out.String()
+		return validateTextContains(childOutput, expectedText)
+	})
+
+	s.Step(`^it prints "([^"]*)" in the terminal$`, func(expectedText string) error {
+		if err = cmd.Wait(); err != nil {
+			return err
+		}
+		childOutput = out.String()
+		return validateTextContains(childOutput, expectedText)
+	})
+
+	s.Step(`^my application contains the file "([^"]*)" with the content:$`, func(filePath string, expectedContent *gherkin.DocString) error {
+		bytes, err := ioutil.ReadFile(path.Join(appDir, filePath))
+		if err != nil {
+			return err
+		}
+		return validateTextContains(strings.TrimSpace(string(bytes)), strings.TrimSpace(expectedContent.Content))
+	})
+
+	s.Step(`^my workspace contains the file "([^"]*)" with content:$`, func(fileName string, expectedContent *gherkin.DocString) error {
+		bytes, err := ioutil.ReadFile(path.Join(appDir, fileName))
+		if err != nil {
+			return err
+		}
+		return validateTextContains(strings.TrimSpace(string(bytes)), strings.TrimSpace(expectedContent.Content))
+	})
+
+	s.Step(`^my workspace contains the empty directory "([^"]*)"`, func(directory string) error {
+		if _, err := os.Stat(path.Join(appDir, directory)); err == nil {
+			return nil
+		}
+		if os.IsNotExist(err) {
+			return fmt.Errorf("%s does not exist", directory)
+		}
+		return nil
+	})
```

You need to return the error here.